Tests/scripts/jira_issues.py

```diff
@@ -62,45 +62,51 @@ def jira_color_text(text: str, color: str) -> str:
     return f"{{color:{color}}}{text}{{color}}"
 
 
+def find_jira_ticket_unresolved_transition(jira_server: JIRA, jira_issue: Issue | None):
+    unresolved_transition = None
+    #  Get the available transitions for the issue
+    transitions = jira_server.transitions(jira_issue)
+    for transition in transitions:
+        if transition['name'] == JIRA_ISSUE_UNRESOLVED_TRANSITION_NAME:
+            unresolved_transition = transition
+            break
+    return unresolved_transition['id'] if unresolved_transition else None
```

Comment:
i know i wrote this initially.
but it can be simplified into a single line with `next(filter(...`